src/pages/Content/index.js

```diff
@@ -1,5 +1,5 @@
 // Global recorded events array
-var recordedEvents = [];
+let recordedEvents = ["#counter-button", "#counter-button", "#counter-button"];
```

Comment:
This should not be here. We are only going to use this for tests, but you don't want it to try to click those slectors on every page...

Reply:
Ok removed the test selectors after testing.